src/integration-builder/initialization.js

```diff
@@ -62,14 +69,15 @@ var initialization = {
                         key = key.replace(/\D/g, '');
                     }
 
-                    if (this.getConsentGroupIds().indexOf(key) > -1) {
-                        boolean = true;
+                    boolean = (this.getConsentGroupIds().indexOf(key) > -1);
+
+                    if (regulation === 'ccpa') {
+                        consent = mParticle.Consent.createCCPAConsent(boolean, Date.now(), consentPurpose, location);
+                        consentState.setCCPAConsentState(consent);
                     } else {
-                        boolean = false;
+                        consent = mParticle.Consent.createGDPRConsent(boolean, Date.now(), consentPurpose, location);
```

Comment:
I know that this is less applicable now, but we may be able to future proof this a little bit by doing a
```
else if regulation === 'gdpr'
```
instead of simply assuming it's `gdpr` in the `else` clause.

For right now there is no difference, but might be easier to read for future developers less familiar with the code.  I am open to keeping it this way though.